src/msha/handlers/dab.handler.spec.ts

```diff
@@ -0,0 +1,89 @@
+import http from "http";
+import { injectHeaders, isDataApiRequest } from "./dab.handler";
+
+describe("isDataApiRequest", () => {
+  const req = { url: "/data-api/users" } as http.IncomingMessage;
```

Comment:
nit: move this `req` inside the method which it is using.. looks like this variable is not being used only at one place.